doc/cephfs/client-auth.rst

```diff
@@ -153,3 +148,104 @@ The optional ``{network/prefix}`` is a standard network name and
 prefix length in CIDR notation (e.g., ``10.3.0.0/16``).  If present,
 the use of this capability is restricted to clients connecting from
 this network.
+
+File system Information Restriction
+===================================
+
+If desired, the monitor cluster can present a limited view of the file systems
+available. In this case, the monitor cluster will only inform clients about
+file systems specified by the administrator. Other file systems will not be
+reported and commands affecting them will fail as if the file systems do
+not exist.
+
+Consider following example. The Ceph cluster has 2 FSs::
+
+    $ ceph fs ls
+    name: cephfs, metadata pool: cephfs_metadata, data pools: [cephfs_data ]
+    name: cephfs2, metadata pool: cephfs2_metadata, data pools: [cephfs2_data ]
+
+But we authorize client ``someuser`` for only one FS::
+
+    $ ceph fs authorize cephfs client.someuser / rw
+    [client.someuser]
+        key = AQAmthpf89M+JhAAiHDYQkMiCq3x+J0n9e8REQ==
+    $ cat ceph.client.someuser.keyring
+    [client.someuser]
+        key = AQAmthpf89M+JhAAiHDYQkMiCq3x+J0n9e8REQ==
+        caps mds = "allow rw fsname=cephfs"
+        caps mon = "allow rw fsname=cephfs"
```

Comment:
The example is definitely correct since I could reproduce the same with `fs authorize` subcommand - 

```
[client.someuser_z]
	key = AQCY5jNfpQImMBAAgu3NyaYSJ/AjkeYC2R06pA==
	caps mds = "allow rw fsname=cephfs2"
	caps mon = "allow rw fsname=cephfs2"
	caps osd = "allow rw tag cephfs data=cephfs2"
```

Reply:
I am bit confused with your comment...

Perhaps, what you mean here is that `fs authorize` should assign `r` instead of `rw`?

Reply:
> Perhaps, what you mean here is that fs authorize should assign r instead of rw?

That's the behavior I would expect. We don't want typical clients to have write access to the FSMap!

Reply:
Yes, I've changed that in last to last push.

Reply:
still not right. No matter what perms we specify for access to the file system, the mon caps should **always** be `r`.

Reply:
The change has been made in the code to enforce MON caps `allow r fsname=<fs-name>`. This is a doc typo.

Reply:
Yes, I made changes in code and elsewhere in doc but somehow this one got missed out. Fixed now - https://github.com/ceph/ceph/pull/32581/files?file-filters%5B%5D=.cc&file-filters%5B%5D=.h&file-filters%5B%5D=.py&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-77943c2916082605baaea0916a21c1d5R176.